packages/bits/src/lib/table/table-virtual-scroll/table-sticky-header.directive.ts

```diff
@@ -179,8 +179,11 @@ export class TableStickyHeaderDirective implements AfterViewInit, OnDestroy {
             const fixedWidth = headColumns[index].classList.contains(FIXED_WIDTH_CLASS);
             if (!fixedWidth) {
                 // Note: Assigning data cell width to the corresponding header column
-                // (using the style width if specified; otherwise, falling back to the offsetWidth)
-                headColumns[index].style.width = cell.style.width || `${cell.offsetWidth}px`;
+                // (using the style width if specified; otherwise, falling back to the clientWidth)
+                const cellWidth = (parseInt(cell.style.width) || cell.clientWidth) -1 + "px";
```

Comment:
do we actually need to change this to clientWidth? if not, we should probably stick with offsetWidth to avoid any surprise bugs with clientWidth.